release/stable/kali-rolling/docker/Dockerfile

```diff
@@ -0,0 +1,80 @@
+# Docker image file that describes an Kali Rolling image with PowerShell installed from tar.gz
```

Comment:
Can we include a copyright, same format as ps1? (not-blocking)

Reply:
- Added the Copyright to the Dockerfile (**Note:** The file has been moved to ([/release/community/stable/kali-rolling/docker/Dockerfile](https://github.com/PowerShell/PowerShell-Docker/blob/92439c1fc923ac8958fb25fdd61f38cb7056d981/release/community-stable/kali-rolling/docker/Dockerfile))